consensus/common.go

```diff
@@ -16,20 +16,3 @@ func subscribeToEvent(evsw types.EventSwitch, receiver, eventID string, chanCap
 	})
 	return ch
 }
-
```

Comment:
These funcs are used in the tests. Maybe they should be moved to common_test.go instead

Reply:
left there. can be moved in later PR